src/Utilities/model.js

```diff
@@ -11,6 +11,7 @@ export function getIssueApplication ({ id }) {
         case 'advisor': return 'Insights';
         case 'ssg': return 'Compliance';
         case 'vulnerabilities': return 'Vulnerability';
+        case 'patch-advisory': return 'Patch';
```

Comment:
it seems to me like it might be a nice follow-up to turn those growing case expressions into constant mappings